location_test.go

```diff
@@ -97,3 +100,47 @@ func TestWithOffsetString(t *testing.T) {
 		}
 	}
 }
+
+func TestGetCurrentLocation(t *testing.T) {
+	specificTz := "Mexico/BajaSur"
+	specificLoc, err := time.LoadLocation(specificTz)
+	if err != nil {
+		t.Fatalf("Cannot initialize Warsaw timezone location")
```

Comment:
Warsaw ???

Reply:
Quite a nice city, have you been there? :)

Reply:
No, I just picked the random timezone which is quite improbable for Snowflake to have an office there ;) So test does not pass accidentally because default location is loaded. Fixed.